webapp/src/hook/hook.js

```diff
@@ -140,7 +140,7 @@ export default class Hooks {
             return (value.public_key === decrypter.PublicKey);
         });
         if (!myMessages || myMessages.length === 0) {
-            return '';
+            return 'Error: Message could not be decrypted!';
```

Comment:
We are just using `Message could not be decrypted!` error message in other places, without `Error:`. Make it consistent return either `Message could not be decrypted!` or `Error: Message could not be decrypted!` everywhere.

Reply:
Done